client/ayon_core/hosts/maya/api/action.py

```diff
@@ -90,7 +90,10 @@ def _update_id_attribute(self, instance, nodes):
             folder_entity = ayon_api.get_folder_by_path(
                 project_name, folder_path, fields={"id"}
             )
-
+            if not folder_entity:
+                self.log.debug(
+                    f"No folder entity found in the project {project_name}")
+                return
```

Comment:
When would this bug occur? This seems like a very dangerous point to get to regardless of the action. An instance with an invalid folder path is not allowed in the new publisher by design - right @iLLiCiTiT ?

Aside of that, if we do need this for whatever reason I'd say this should also log the folder path it tried to find.

Something like: `Folder {folder_path} not found in project {project_name}`

Reply:
> When would this bug occur?

Same question?

If it can somehow happen (which should not), shouldn't the action actually fail in that case as it seems like "all fixed" but on next publish it will fail again?